Divergic.Logging.Xunit/ScopeWriter.cs

```diff
@@ -52,7 +52,8 @@ public void Dispose()
 
         private static string BuildPadding(int depth)
         {
-            return new string(' ', depth * DefaultFormatter.PaddingSpaces);
+            var config = new LoggingConfig();
```

Comment:
This produces a tight coupling between the ScopeWriter and the LoggingConfig class. The other issue here is that it ignores any custom config object that the consumer was wanting to use. This parameter needs to be passed into the constructor of the ScopeWriter.

Reply:
BuildPadding(int depth) seems like it should be moved to the "LoggingProcessor" class.